app/page.tsx

```diff
@@ -9,14 +9,39 @@ export default function Page() {
       <PageTitle>Department of Decentralization</PageTitle>
       <div className="prose max-w-none pb-8 pt-8 dark:prose-invert">
         <div>
-          The Department of Decentralization is a collective of people from various crypto,
-          decentralization and peer-to-peer communities in and around Berlin. The group assembled in
-          2018 to organize ETHBerlin and has been active since.
+          The Department of Decentralization is a collective of people from the cryptocurrency,
+          decentralization, and peer-to-peer community in Berlin. The group assembled in 2018 to
+          organize the{' '}
+          <a href="https://ethberlin.com" target="_blank" rel="noopener noreferrer">
+            ETHBerlin
+          </a>{' '}
+          hackathon and has been active since.
         </div>
         <div>
           We aim to be an agnostic vehicle to drive adoption, educate newcomers, and raise awareness
           of the challenges and benefits of decentralization and open-source software.
         </div>
+        <div>
+          All our events are free to attend and we try, whenever possible, to deliver a
+          distraction-free experience by not hosting sponsors, paid talks, or any other commercial
+          components. Our collective entirely runs on donations.
+        </div>
+        <div>
```

Comment:
I like it and hope we can keep it that way. But we need to have a way to automatically revisit this statement when times change. Also wondering if "All our events" implies also past events.
Maybe also another wording can help being more correct and future proof here?

Reply:
> Also wondering if "All our events" implies also past events.

Yes, this also applies to past events. They have always been free.

I see that we need to revisit this in future if we want to change this policy but I really hope not.